components/ui/PrintButton.js

```diff
@@ -0,0 +1,22 @@
+
+import React, { useState } from 'react';
+
+const PrintButton = () => {
+  const print = () => {
+    setTimeout(function () {
+      window.print();
+    }, 500);
+  }
+
+  return(
+    <>
+      <div className="printi flex flex-col items-center invisible md:visible">
```

Comment:
This is the proper way of hiding elements using Tailwind and toggling them back. Update the same.
`      <div className="flex-col items-center hidden md:flex">
`